src/test/java/com/spotify/docker/client/DockerHostTest.java

```diff
@@ -98,7 +98,7 @@ public void testCertPathFromEnv() throws Exception {
     DockerHost.setSystemDelegate(systemDelegate);
 
     assertThat(DockerHost.certPathFromEnv(), equalTo("foo"));
-    assertThat(DockerHost.certPathFromEnv(), equalTo("bar/.docker"));
+    assertThat(DockerHost.certPathFromEnv(), equalTo(null));
```

Comment:
Yes, as @johnflavin said, I wanted to test different return values based on different return values from the mock. Thanks for mentioning `nullValue()` though.